spec/requests/api/v1/topics/index_spec.rb

```diff
@@ -3,40 +3,65 @@
 describe 'GET api/v1/topics', type: :request do
   let(:user) { create(:user) }
   let!(:headers) { auth_headers }
-  let!(:topics) { create_list(:topic, 3) }
 
   subject { get api_v1_topics_path, headers: headers, as: :json }
-  before { subject }
-
   context 'when there are topics' do
+    let!(:topics) { create_list(:topic, 3, :with_image) }
+
     it 'returns a successful response' do
+      subject
+
       expect(response).to have_http_status(:success)
     end
 
     it 'returns the correct amount of topics' do
+      subject
+
       expect(json[:topics].size).to eq(3)
     end
 
     it 'returns all the topics' do
       subject
+
       expect(json[:topics].pluck(:id, :name)).to match_array(topics.pluck(:id, :name))
     end
+
+    it 'returns the image in the body response' do
+      subject
+      json[:topics].each do |topic|
```

Comment:
I'm not 100% sure about using an each loop in the specs, I'd prefer some other kind of approach, but it's ok